packages/aws-cdk-lib/aws-dynamodb/lib/table.ts

```diff
@@ -1586,6 +1620,17 @@ export class Table extends TableBase {
         : undefined);
   }
 
+  private validateCCI (props: IContributorInsightsConfigurable): ContributorInsightsSpecification | undefined {
+    if (props.contributorInsightsSpecification !==undefined && props.contributorInsightsEnabled !== undefined) {
```

Comment:
This looks very similar to the validateCCI function in table-v2. Can we combine them in the shared file as a utility? They essentially do the same thing, and you would need to parameterize the old and new names.

Also, looks like code here might not be linted?